src/alexapi/triggers/pocketsphinxtrigger.py

```diff
@@ -43,7 +43,11 @@ def setup(self):
 
 		# Hide the VERY verbose logging information when not in debug
 		if logging.getLogger('alexapi').getEffectiveLevel() != logging.DEBUG:
-			ps_config.set_string('-logfn', '/dev/null')
+
+			if self._config['platform']['device'] == 'windows':
```

Comment:
Detection as above and do something like 

```
null_path = 'nul' if platform.system() == 'Windows' else '/dev/null'
ps_config.set_string('-logfn', null_path)
```
Because you want to _fork_ the parameter, not the actual function / code.